src/applications/caregivers/components/FormFields/FacilityList.jsx

```diff
@@ -11,20 +11,17 @@ const FacilityList = props => {
   const reviewMode = props?.formContext?.reviewMode || false;
   const submitted = props?.formContext?.submitted || false;
 
-  const [dirty, setDirty] = useState(false);
-
   const handleChange = e => {
     onChange(e.detail.value);
-    setDirty(true);
   };
 
   const showError = () => {
     if (error) {
       return error;
     }
 
-    return (submitted || dirty) && !value
-      ? content['validation-facialities--default-required']
```

Comment:
This logic was causing it to flash the error message after selecting a value on the list. The parent component mostly handles errors for this component, since now it passes an error and the update callback so it felt unnecessary to have this here.